src/core/public/index.ts

```diff
@@ -167,7 +167,16 @@ export {
   IHttpResponseInterceptorOverrides,
 } from './http';
 
-export { OverlayStart, OverlayBannersStart, OverlayRef } from './overlays';
+export {
+  OverlayStart,
+  OverlayBannersStart,
+  OverlayRef,
+  OverlayFlyoutStart,
+  OverlayFlyoutOpenOptions,
+  OverlayModalOpenOptions,
+  OverlayModalConfirmOptions,
+  OverlayModalStart,
+} from './overlays';
```

Comment:
There was a lot of missing exports from the `overlays` package (and a lot of missing associated generated doc files) I saw that when adding the `maxWidth` option did not update any file from the generated doc, so I fixed it in this PR.